### DISCUSSION TITLE
New account status - block features except earning Achievements

### DISCUSSION THREAD

**Discussion by SporyTike** [RetroAchievements/RAWeb#777] [Ideas]
I am not a real staff member on the site but Iike the title said I would request a new account status. In the last few days there was a decision for a user to get a permanently ban because he annoyed the community. I discussed with the user in private and found out that a big problem with the situation is that he is completely banned to even earn achievements anymore. I would like to have an extra account status (like "Banned", "Untracked", ...) which can be used in specific cases like this.
The status should block text-based features like forum topic, private messages and comments but not the possibility to login in the emulator and earn achievements. Maybe also allow the user to write on the own user page comments so he can at least react to other posts and send private messages to friends on the website.

To have a good overview a list of the wished features of the new status:
- Disable all features like "Banned" (except ones listed below)
- Able to earn achievements
- Able to use the set request feature
- Able to use the game ranking feature

Wished features if possible:
- Able to comment on the own game page
- Write private messages to friends
- Create tickets
- Able to accept friend requests (not sending requests)

**Comment by kdecks** [comment]
"Muted", (on site) is a planned feature for v2.. well I can't confirm it, but we talked about it.

It had most of those concepts, but you def are fleshing it out.

**Comment by meleu** [comment]
I agree with what the title says but disagree with the @SporyTike's description.

IMO such user status should be "earn achievements only" and strictly that. With no possibility to communicate to anyone but admins/mods.

- no set requests
- no friend requests
- no comments anywhere
- no tickets
- unable to set a motto
- user wall disabled
- no private messages (only allowed to communicate with admins/mods)
- always untracked

In my mind such status means "you're still allowed to play retrogames with achievements, but you're unable to be part of the community".
